notebooks/118-optimize-preprocessing/118-optimize-preprocessing.ipynb

```diff
@@ -0,0 +1,537 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    if Path(MODEL_DIR, f'{model_name}.prototxt').exists() and Path(MODEL_DIR, f'{model_name}.caffemodel').exists():
</pre><p>this check unneccesary, download_file writes the same message if file exists</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/811/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>